onlinejudge_verify/languages/nim.py

```diff
@@ -0,0 +1,77 @@
+# Python Version: 3.x
+import pathlib
+import shlex
+import subprocess
+from logging import getLogger
+from typing import *
+
+from onlinejudge_verify.config import get_config
+from onlinejudge_verify.languages.models import Language, LanguageEnvironment
+
+logger = getLogger(__name__)
+
+
+class NimLanguageEnvironment(LanguageEnvironment):
+    NIMFLAGS: List[str]
+    CompileTo: str
```

Comment:
```suggestion
    compile_to: str
```

`NIMFLAGS` はそういう名前の変数なのでしかたがないですが、そうでもない普通の変数は snake_case で書いてください。

Reply:
nim.pyでの変数表記はsnake_caseにするとして、こちらのようにコマンドのオプションがCompileToC, CompileToCppなどとなっているのですが、tomlのconfigとしてはCompileToのままにしますか？それともこちらもcompile_toにしますか？

Reply:
リンクを貼るのを忘れていました。こちらです。

https://nim-lang.org/docs/nimc.html

Reply:
> こちらのようにコマンドのオプションがCompileToC, CompileToCppなどとなっているのですが

なってなさそうに見えます。
`compileToC` や `compileToCpp` は見つかりますが、`CompileToC` や `CompileToCpp` はありません。

Reply:
あ、すみません。見間違えました。ではtomlの変数名はcompileToにしますか？それとも、nim.pyと統一してcompile_toにしますか？

Reply:
ところで `compileToC` や `compileToCpp` を使った場合をテストしたいことはあるのですか？ 非 Nim ユーザとしては、常に `compile` を使えば十分そうに見えます。

Reply:
そうですね。atcoderの新しい言語アップデートではcompileToCppになる予定で、compileだとcompileToCになるみたいなのです。C++がCを兼ねているので大体compileToCppで事足りるのですが、一方でポインタ周りの処理でcompileToCでないとできないことがたまにあるようです。どうしましょうか？

Reply:
- 標準は `compileToC` だが AtCoder は `compileToCpp` になってるということなら、両方を切り替えられるようにしておきたいというのは分かります。設定できるようにしておきましょう。
- 変数名は `compile_to` にしてください。ただし、これは `nim.py` と統一するため**ではなく**、TOMLの命名規則がたいてい camelCase だからです。